### PR TITLE
Update nebula config reference

### PR DESCRIPTION
**Pull request by jasikpark**
Related to https://github.com/DefinedNet/nebula-docs/issues/19

You can see the deploy preview in `www`

It'd be helpful to get some confirmation on which things I've labeled as reloadable and not reloadable

### CODE REVIEW

File: src/data/config-reference.md [hunk 1]
```diff
@@ -204,8 +204,8 @@ en:
         - name: port
           default: 0
           description: >
-            port is the UDP port nebula should use on a host. setting this to 0
-            will dynamically assign a port number.
+            port is the UDP port nebula should use on a host. setting this to `0`
```
Comment (IanVS) [inline src/data/config-reference.md:207]: If the default is `0`, does "setting this to `0`" really make a difference?  Or, should we say that leaving it at the default dynamically....

Comment (JohnMaguire) [reply src/data/config-reference.md:207]: This is the existing text that I believe comes originally from the `config.yml`. "setting this to `0`" makes sense to me, and makes it clear what happens if you explicitly set it, e.g. through an ansible playbook

```
port: {{ port | default(0) }}
```

Comment (IanVS) [reply src/data/config-reference.md:230]: I would prefer language like `A value of "0" (the default) will dynamically assign a port number.`

As it is, it makes it sound like the default value is not to dynamically assign a port number, and that setting the value will cause that to happen.  But, I'm 1/5, feel free to ignore.

**Comment by IanVS** [comment]
You'll need to change the CMS config as well, to expose these settings.

**Comment by jasikpark** [comment]
> You'll need to change the CMS config as well, to expose these settings.

Mind pairing w/ me on this? I'm not sure how or where I'd pull them in

**Review by IanVS — CHANGES_REQUESTED** [review]
I think we might want to find a way to explain what `reloadable` means, and we should definitely get a review on the content from @nbrownus.  But I only had one blocker here.

File: src/components/ConfigReference.astro [hunk 2]
```diff
@@ -35,6 +37,8 @@ const configOptions = options[locale] ? options[locale].options : fallbackOption
         
         return (<ConfigOption
           isRequired={suboption.required}
+          isReloadable={option.reloadable}
+          defaultValue={option.default}
```
[the remark this answers is not in the document]
Comment (jasikpark) [reply src/components/ConfigReference.astro:41]: oops, I was copying over from the `www` PR

**Comment by jasikpark** [comment]
Yeah, I don't fully understand what sending `SIGHUP` would mean exactly... I guess it's as much as:

> While nebula is running for `RELOADABLE` properties you can edit the values in the config and signal the running nebula process with `SIGHUP` to have nebula update those config properties.
> ex:
> run `ps aux | grep nebula` to find nebula's PID & then run `kill -s HUP $PID`

**Review by IanVS — APPROVED** [review]

**Comment by jasikpark** [comment]
@nbrownus do you think this could be merged if we yoink out the `required` pills for the moment and only add the `default` pills + the updated docs?

**Comment by nbrownus** [comment]
I will need to re-review the copy but in general yes, I think we need to workshop the reloadable indicators more.

**Comment by jasikpark** [comment]
> I will need to re-review the copy but in general yes, I think we need to workshop the reloadable indicators more.

Ok, I'll try to push up a commit that removes the "reloadable" indicators before too long

**Comment by jasikpark** [comment]
Replaced by #42

**Pull request closed without merging**